src/config/dynamic-theme-fixes.config

```diff
@@ -17582,6 +17582,8 @@ INVERT
 .MathJax:not(span#MathJax_Zoom > .MathJax)
 span#MathJax_Zoom
 .mw-ext-score
+.mw-logo
```

Comment:
So, invert the wordmark and tagline but keep the globe as it is, correct?

Reply:
![Screenshot 2022-03-11 at 23-51-16 Wikipedia the free encyclopedia](https://user-images.githubusercontent.com/10338703/157911549-de269341-95ed-49cb-9894-491cb79adc13.png)

Reply:
Hmm, wait a second. I think I can add .mw-logo-icon to invert the globe

Reply:
![Screenshot 2022-03-12 at 00-08-53 Wikipedia the free encyclopedia](https://user-images.githubusercontent.com/10338703/157914496-9d69c4d3-57c9-4141-ba0c-3e033a55845a.png)

Reply:
Why invert the globe?

Reply:
Because the (old) Vector has the inverted globe

https://en.wikipedia.org/w/index.php?title=Main_Page&useskin=vector

![image](https://user-images.githubusercontent.com/10338703/157921509-e19d87a5-e2aa-4116-ae03-094497d690b2.png)

Although, the normal globe looks fine for me

Reply:
> Because the (old) Vector has the inverted globe (default skin on English Wikipedia)

Yeah... that's because the globe + text was 1 picture.

Reply:
Alright, I will remove the globe inversion

Reply:
![image](https://user-images.githubusercontent.com/10338703/157923463-6d390a75-9abb-4000-b774-1f288b8ea72e.png)

Reply:
Just wordmark

![image](https://user-images.githubusercontent.com/10338703/158002309-1bb0d7bf-c18f-43e8-9d17-3e75779f715e.png)

Reply:
Related to #8427, sidebar button inversion is not needed so I modified the patch